<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallInOutDialog</class>
 <widget class="QDialog" name="CallInOutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Call in/out Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>351</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Please input the method to get the call in/out methods</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>531</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Example:Lcom/dlp/SMSReplicatorSecret/ShadyDB; (Landroid/content/Context;)V,&lt;init&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>551</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Class Name:  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_class">
      <property name="text">
       <string>Lcom/dlp/SMSReplicatorSecret/ShadyDB;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>551</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Method Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_method">
      <property name="text">
       <string>&lt;init&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>551</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Descriptor:    </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_descriptor">
      <property name="text">
       <string>(Landroid/content/Context;)V</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QCheckBox" name="checkBox_in">
      <property name="text">
       <string>Call in</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_out">
      <property name="text">
       <string>Call out</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>461</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked(bool)</signal>
   <receiver>CallInOutDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked(bool)</signal>
   <receiver>CallInOutDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
